src/lib/vm.ts

```diff
@@ -126,7 +126,7 @@ export async function registerVM(program: Command) {
   vm.command("logs")
     .description("View or tail VM logs")
     .option("-i, --instance <id>", "Filter logs by instance ID")
-    .option("-l, --limit <number>", "Number of log lines to fetch", "50")
+    .option("-l, --limit <number>", "Number of log lines to fetch", "100")
```

Comment:
Bit confused on why the default value/the type of this option isn't `number`

Reply:
it's passed in as a query param which requires a string. 

the api endpoint then does the validation. 

However, I agree that it should be an integer and we should validate it as early as possible. 

made the change so it's a number now.